packages/flame_texturepacker/lib/src/texture_packer_atlas.dart

```diff
@@ -25,18 +25,22 @@ class TexturePackerAtlas {
   /// If [useOriginalSize] is true, the sprites loaded will be use original size
   /// instead of the packed size. For animation sprites, load with origin size
   /// is recommended for smooth result.
+  ///
+  /// If [prefix] is not specified, the atlas file will be expected to under
+  /// a folder named "images" under [Flame.assets]
```

Comment:
```suggestion
  /// If [prefix] is not specified, the atlas file [path] will be resolved
  /// relative to the `assets/images/` directory.
```
Since `Flame.assets` is an object I think the updated version will read a bit clearer.